tutorials/general/t8_step4_partition_balance_ghost.cxx

```diff
@@ -149,40 +149,19 @@ t8_step4_partition_ghost (t8_forest_t forest)
    * We currently support ghost mode T8_GHOST_FACES that creates face neighbor ghost elements
    * and will in future also support other modes for edge/vertex neighbor ghost elements.
    */
-  t8_forest_set_ghost (new_forest, 1, T8_GHOST_FACES);
+  t8_forest_set_ghost (new_forest, 1, T8_GHOST_VERTICES);
   /* Commit the forest, this step will perform the partitioning and ghost layer creation. */
   t8_forest_commit (new_forest);
 
   return new_forest;
 }
 
-/* In this function we adapt a forest as in step3 and balance it. 
- * In our main program the input forest is already adapted and then the resulting twice adapted forest will be unbalanced.
- */
-static t8_forest_t
-t8_step4_balance (t8_forest_t forest)
+int
```

Comment:
why is balance removed?
Maybe it is a good idea to make an exampe where one can compare vertex and face ghosts?

Reply:
Because the changes in the tutorial were not meant to be reviewed or merged, rather to test stuff out locally.

This feature is not tested at all, and I see that as a bigger priority than fixing tutorials